Cargo.toml

```diff
@@ -187,6 +187,11 @@ panic = "unwind"
 strip = "symbols"
 overflow-checks = true
 
+[profile.dev]
+opt-level = "z"
+panic = "unwind"
+strip = "symbols"
+overflow-checks = true
```

Comment:
No, I need to work out how to make a local configuration for this so I don't keep accidentally committing it, but need it to test on a raspberry pi in seconds rather than minutes